lab-steven/routes/routes.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+// const Seahawk = require('../model/seahawk');
+const hawkCtrlr = require('../controller/hawk-controller');
+
+module.exports = function(router){
+  router.get('/api/hawk/:id', (req, res) => {
+    hawkCtrlr.readHawk({ '_id' : `${req.params.id}`})
+    .then(data => res.json(data))
+    .catch(err => res.status(404).send(err.message));
+  });
+
+//getall
+  router.get('/api/hawk', (req, res) => {
+    hawkCtrlr.readHawk({})
+    .then(data => res.json(data))
+    .catch(err => res.status(404).send(err.message));
+  });
+
+  router.post('/api/hawk', (req, res) => {
+    hawkCtrlr.createHawk(req.body)
+    .then(data => res.json(data))
+    .catch(err => res.status(400).send(err.message));
+  });
+
+  router.put('/api/hawk/:id', (req, res) => {
+    hawkCtrlr.updateHawk({ '_id' : `${req.params.id}`}, req.body)
+    .then(data => res.json(data))
```

Comment:
your updated seahawk is updating on mongo, but not being sent out correctly. it's either here, or in the route itself, but the returned object looks weird. please address!